pkg/api/v1/types.go

```diff
@@ -1022,11 +1022,16 @@ type Container struct {
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/containers.md#containers-and-commands
 	Args []string `json:"args,omitempty"`
 	// Container's working directory.
-	// Defaults to Docker's default. D
-	// efaults to image's default.
+	// Defaults to Docker's default, which may be specified by the container
```

Comment:
I'm trying to reserve the term "Default" for the way we explicitly initialize fields during the defaulting pass.

If the field will remain empty if unspecified, I've recently suggested "inferred to be...". In this case, that doesn't work perfectly.

How about: "If unspecified, uses container runtime's default, which may be specified by the container"